psi4/src/read_options.cc

```diff
@@ -1720,7 +1744,7 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_double("CPHF_MEM_SAFETY_FACTOR", 0.75);
         /*- SCF Type
          -*/
-        options.add_str("SCF_TYPE", "DIRECT", "DIRECT DF PK OUT_OF_CORE PS INDEPENDENT GTFOCK");
+        options.add_str("SCF_TYPE", "DIRECT", "DIRECT DF PK OUT_OF_CORE PS INDEPENDENT GTFOCK COSX");
```

Comment:
This is probably a @loriab question, but why do we have a CPHF method at all, rather than `properties('scf')`? If we removed that, this option duplication becomes unnecessary.

Reply:
I'm not sure. The CPHF block feeds libfock but so do other blocks. There is `properties("scf")`, but mostly ppl run oeprop directly instead.